hooks/create_move/aimbot.c

```diff
@@ -255,10 +293,16 @@ void projectile_aimbot(void *localplayer, struct user_cmd *user_cmd, int weapon_
         esp_projectile_pos = projectile_target_pos;
         esp_predicted_time = get_global_vars_curtime() + target_predicted_time;
     }
+
+    movement_fix(user_cmd, original_view_angle, original_forward_move, original_side_move);
```

Comment:
Does the projectile aimbot need a movement fix? If so you'll need to add the `original_view_angle`, `original_forward_move`, and `original_side_move` variables to this function.

Reply:
From usage, It may not be necessary. There aren't any fully automatic projectile weapons, besides maybe the Air Strike, but that would just need some testing. Regardless, oh goodness that was a silly mistake, I'll fix it.